skyvern-frontend/package-lock.json

```diff
@@ -73,6 +73,7 @@
       "version": "5.2.0",
       "resolved": "https://registry.npmjs.org/@alloc/quick-lru/-/quick-lru-5.2.0.tgz",
       "integrity": "sha512-UrcABB+4bUrFABwbluTIBErXwvbsU/V7TZWfmbgJfbkwiBuziS9gxdODUyuiecfdGQ85jglMW6juS3+z5TsKLw==",
+      "dev": true,
```

Comment:
We shouldn't have any changes in this file. Please revert them.

Reply:
these are not changes it's already in the main directory please chek